### PR TITLE
updated mocha version and fixed relevant files

### PR DESCRIPTION
**Pull request by makambalaji**
### What does this PR do?
This pr consists of mocha version changes from `7.1.1` to `9.1.3` in package.json for fixing npm versions w.r.to audit.
Updated the opts deprecated files with cong files in json format. To make sure the consistency, 
Created config files with the same names which used earlier
It is also part of #20864 

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix or new feature ](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix-or-new-feature)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by dmytro-ndp — APPROVED** [review]
Happy path tests passed https://github.com/eclipse/che/pull/20892#issuecomment-987723414

@makambalaji : please, create follow up PR with cherry picked changes into the 7.40.x branch.

**Review by ScrewTSW — APPROVED** [review]
Great work! LGTM

**Comment by makambalaji** [comment]
> Happy path tests passed [#20892 (comment)](https://github.com/eclipse/che/pull/20892#issuecomment-987723414)
> 
> @makambalaji : please, create follow up PR with cherry picked changes into the 7.40.x branch.

Created a cherry pick for 7.40.x version : https://github.com/eclipse/che/pull/20896/

**Comment by dmytro-ndp** [comment]
@makambalaji: looks like [recent changes](https://github.com/eclipse/che/pull/20892/commits/6e1289997a8961f7a4ac3c637401c9125dbf8beb) have broken Happy path tests https://main-jenkins-csb-crwqe.apps.ocp4.prod.psi.redhat.com/job/Che/job/e2e/job/minikube/job/pr-check/job/che-tests-pr-check/298/console

**Review by dmytro-ndp — CHANGES_REQUESTED** [review]
@makambalaji: looks like [recent changes](https://github.com/eclipse/che/pull/20892/commits/6e1289997a8961f7a4ac3c637401c9125dbf8beb) have broken Happy path tests https://main-jenkins-csb-crwqe.apps.ocp4.prod.psi.redhat.com/job/Che/job/e2e/job/minikube/job/pr-check/job/che-tests-pr-check/298/console :
```
 ​> @eclipse-che/che-e2e@7.41.0-SNAPSHOT sc /tmp/e2e
​>tsc -p .
 ​
 ​utils/requestHandlers/CheApiRequestHandler.ts(28,21): error TS2532: Object is possibly undefined'.
 ​utils/requestHandlers/CheApiRequestHandler.ts(50,21): error TS2532: Object is possibly undefined'.
 ​npm ERR! cod ELIFECYCLE
 ​npm RR! errno 2
 ​npm ERR! @eclipse-che/che-e2e@7.41.0-SNAPSHOT tsc `tsc -p .`
 ​npm ERR! Eit status 2 ​npm ERR! 
 ​npm ERR! Failed at the @eclipse-che/che-e2e@7.41.0-SNAPSHOTtsc script.
 ​npm ERR! This is probably not a problem with npm. There is likely additional logging otpt above.
 ```
 
 Take a look, please.

File: tests/e2e/package.json [hunk 1]
```diff
@@ -37,10 +37,10 @@
     "@types/node": "11.13.4",
     "@types/rimraf": "2.0.2",
     "@types/selenium-webdriver": "3.0.16",
-    "axios": "0.21.2",
```
Comment (dmytro-ndp) [inline tests/e2e/package.json:40]: There is no information about an update of axios in PR description.

What was the reason to upgrade the version from `0.21.2` to `0.24.0`?

It's `0.21.1` downstream in CRW tests.

Comment (makambalaji) [reply tests/e2e/package.json:40]: Its updated as part of npm audit fix, but it doesn't have any major feature changes. On re-running audit-fix command, this package version is not getting updated. Let me revert this change. Maybe later we can think about upgrading the packages.

Comment (makambalaji) [reply tests/e2e/package.json:40]: > @makambalaji: looks like [recent changes](https://github.com/eclipse/che/pull/20892/commits/6e1289997a8961f7a4ac3c637401c9125dbf8beb) have broken Happy path tests https://main-jenkins-csb-crwqe.apps.ocp4.prod.psi.redhat.com/job/Che/job/e2e/job/minikube/job/pr-check/job/che-tests-pr-check/298/console :
> 
> ```
>  ​> @eclipse-che/che-e2e@7.41.0-SNAPSHOT sc /tmp/e2e
> ​>tsc -p .
>  ​
>  ​utils/requestHandlers/CheApiRequestHandler.ts(28,21): error TS2532: Object is possibly undefined'.
>  ​utils/requestHandlers/CheApiRequestHandler.ts(50,21): error TS2532: Object is possibly undefined'.
>  ​npm ERR! cod ELIFECYCLE
>  ​npm RR! errno 2
>  ​npm ERR! @eclipse-che/che-e2e@7.41.0-SNAPSHOT tsc `tsc -p .`
>  ​npm ERR! Eit status 2 ​npm ERR! 
>  ​npm ERR! Failed at the @eclipse-che/che-e2e@7.41.0-SNAPSHOTtsc script.
>  ​npm ERR! This is probably not a problem with npm. There is likely additional logging otpt above.
> ```
> 
> Take a look, please.

Removed the axios version upgrade, looks like that is the issue related to this error

**Comment by makambalaji** [comment]
[crw-ci-test]

File: tests/e2e/package.json (hunk 1 above)
Comment (dmytro-ndp) [reply tests/e2e/package.json:40]: Thank you!

**Comment by dmytro-ndp** [comment]
[crw-ci-test]

**Review by dmytro-ndp — APPROVED** [review]

**Comment by makambalaji** [comment]
[crw-ci-test]

**Pull request merged**